sdk/storage/azure-storage-blob/HISTORY.md

```diff
@@ -1,6 +1,15 @@
 # Change Log azure-storage-blob
 
-## 12.1.0b1 Unreleased
+## 12.1.0
+
+**New features**
+- Added `download_blob` method to the `container_client`.
+- All the clients now have a `close()` method to close the sockets opened by the client.
```

Comment:
For clarity, I would add a note to say that if you use a context manager it's not necessary to call this method.

Reply:
And make sure the docstring of close says it too.